mypy/semanal.py

```diff
@@ -3546,6 +3550,11 @@ def visit_file(self, file: MypyFile, fnam: str, mod_id: str, options: Options) -
                 else:
                     typ = UnionType([UnboundType('__builtins__.str'),
                                      UnboundType('__builtins__.unicode')])
+            elif name == '__path__':
+                # __path__ is only defined on __init__.py files
+                if not os.path.basename(fnam).lower() == '__init__.py':
+                    continue
+                typ = UnboundType(t)
```

Comment:
Hm, I don't know the answer to that. Maybe @JukkaL or @ilevkivskyi knows? I think you're pretty close with `UnboundType('__builtins__.list', args=[UnboundType('__builtins__.str')])` and maybe we just need to find a way to suppress that error.